modules/formulize/include/synchronization.php

```diff
@@ -0,0 +1,226 @@
+<html>
+    <head><title>Test</title></head>
+    <body>
+    <?php
+        include "../../../mainfile.php";
+        include "../class/tableInfo.php";
+
+        /*
+         * TO DO:
+         *          
+         *      1. add folder stucture to template file part of archive
+         */
+        
+        /*
+         * doExport function exports template files and current Formulize database state to a ".zip" archive
+         * 
+         * param archiveName        string representing path to new or existing zip file. path must have ".zip" extension
+         */
+        function doExport($archiveName){
```

Comment:
Makes sense to have doExport receive a preferred filename, also might make sense for this to have an optional parameter for the folder, but default to the Export folder if nothing specified.

For that matter, might make sense for the filename to also be optional, and default inside the function to something sensible if no filename is passed in.